node/rpc-client/src/wallet_ops.rs

```diff
@@ -14,9 +14,10 @@ pub async fn wallet_new(
 }
 
 pub async fn wallet_default_address(
+    params: WalletDefaultAddressParams,
```

Comment:
Are you adding this parameter for consistency?

Reply:
Not in the first place, it is needed for the `bind_func` macro (to avoid all the VM binding boilerplate code).